src/app/Sidebar/RightSidebar.js

```diff
@@ -18,41 +18,24 @@ import SignUp from '../../components/Sidebar/SignUp';
   state => ({
     authenticated: getIsAuthenticated(state),
     authenticatedUser: getAuthenticatedUser(state),
-    followingList: getFollowingList(state),
-  }),
+    recommendations: getRecommendations(state),
+  }), {
+    updateRecommendations,
+  },
 )
 export default class RightSidebar extends React.Component {
   static propTypes = {
     authenticated: PropTypes.bool.isRequired,
     authenticatedUser: PropTypes.shape().isRequired,
-    followingList: PropTypes.arrayOf(PropTypes.string).isRequired,
+    recommendations: PropTypes.arrayOf(PropTypes.shape({ name: PropTypes.string })).isRequired,
+    updateRecommendations: PropTypes.func.isRequired,
```

Comment:
We tend not to use required functions as props. Can you make it not required and set NOOP (`() => {}`) as `defaultProp` for it?